src/api.js

```diff
@@ -6,12 +6,18 @@ module.exports = class DanceAPI {
       setBackground: color => {
         nativeAPI.setBackground(color.toString());
       },
-      setBackgroundEffect: effect => {
-        nativeAPI.setBackgroundEffect(effect.toString());
+      setBackgroundEffect: (effect, palette) => {
+        nativeAPI.setBackgroundEffect(effect.toString(), palette);
+      },
+      setBackgroundEffectWithPalette: (effect, palette) => {
+        nativeAPI.setBackgroundEffect(effect.toString(), palette);
       },
       setForegroundEffect: effect => {
         nativeAPI.setForegroundEffect(effect.toString());
       },
+      setForegroundEffectExtended: (effect) => {
```

Comment:
Do we need this new function in the API if the signature / behavior is the same? Can't both the old and new blocks call the same existing function ?

Reply:
@ryansloan mentioned there was some reason we can't have two different blocks that point to the same function.  I don't know the details, but providing an alias function in the API seemed like a good workaround.

I'm not actually sure what that limitation is - we use the full block name when serializing to XML (`<block type="Dancelab_setBackgroundEffect">`), so there must be some other place we depend on the `func` being unique across all loaded blocks.

Reply:
The blocks editor enforces uniqueness, I'm not sure how much of that is just the editor vs. some other underlying requirement